packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -31,15 +32,14 @@ function getHtml(el) {
 	return `${DOCTYPE}${htmlMarkup}`;
 }
 
-export function getRedirect(context) {
+export function getRedirect(context: { url?: string, permanent?: boolean }) {
 	if (!context || !context.url) {
 		return;
 	}
 	// use `URL` to ensure valid character encoding (e.g. escaped emoji)
-	const isFragment = context.url.startsWith('/');
-	const urlToFormat = isFragment
-		? `${DUMMY_DOMAIN}${context.url}`
-		: context.url;
+	const url: string = context.url;
+	const isFragment = url.startsWith('/');
+	const urlToFormat = isFragment ? `${DUMMY_DOMAIN}${url}` : url;
```

Comment:
Fixes a Flow error that `context.url` cannot be cast to string because it can be undefined. I've also added a few [test cases](https://github.com/meetup/meetup-web-platform/pull/583/files#diff-d9666b70703fe4e60a7d86d081316807) to validate the behavior when `context.url` is not defined or empty